addons/sale/static/src/js/product_list/product_list.scss

```diff
@@ -0,0 +1,25 @@
+.quantity_alignment {
+    @media (max-width: 768px) {
+        text-align: end;
+    }
+
+    @media (min-width: 769px) {
+        text-align: center;
+    }
+}
+
+.fixed_border_for_block_display {
+    border-bottom: solid;
+    border-bottom-width: 1px;
+    border-color: inherit;
```

Comment:
I had to override the tr styling because it would appear like this otherwise
![2024-12-10_09-53](https://github.com/user-attachments/assets/1b06c4b8-f0ff-4ba9-8a87-18f10f619ce7)